Source/ModCompendiumLibrary/ModSystem/Builders/IModBuilder.cs

```diff
@@ -9,7 +9,8 @@ public interface IModBuilder
         /// </summary>
         /// <param name="root">Virtual directory containing mod file data.</param>
         /// <param name="hostOutputPath">Optional. If specified, the mod builder can be expected to output the built mod file to the location specified.</param>
+        /// <param name="compression">Optional. If compression == "True", the mod builder will compress the resulting cpk.</param>
         /// <returns></returns>
-        VirtualFileSystemEntry Build( VirtualDirectory root, string hostOutputPath = null );
+        VirtualFileSystemEntry Build( VirtualDirectory root, string hostOutputPath = null, string compression = null );
```

Comment:
Compression should be a bool. convert it to a string with .ToString()
Also change the name to `useCompression`. 
I like booleans to be prefixed with a verb as they represent state and it reads a bit more naturally.